Makefile

```diff
@@ -100,11 +100,17 @@ deploy-eth: compile-eth
 	npx hardhat run --network $(NETWORK) scripts/2_deploy_bridge_contracts.js
 
 .PHONY: deploy-eth-live
-deploy-eth-live: # Deploy only the MOST contract on a live ethrereum network (tesnet or mainnet)
+deploy-eth-live: # Deploy only the MOST contract on a live ethereum network (testnet or mainnet)
 deploy-eth-live: compile-eth
 	cd eth && \
 	npx hardhat run --network $(NETWORK) scripts/deploy_bridge_live.js
 
+.PHONY: upgrade-eth-live
+upgrade-eth-live: # Upgrade the MOST contract on a live ethereum network (testnet or mainnet)
+upgrade-eth-live: compile-eth
+	cd eth && \
+	npx hardhat run --network $(NETWORK) scripts/upgrade_bridge_live.js
```

Comment:
Would use "upload code" or something like this instead of "upgrade" as the script itself does not perform an upgrade